### PR TITLE
#56 tidy scripts dependencies

### PR DESCRIPTION
**Pull request by wlthomson**
A few fixes and tidyups as mentioned in #56. 

Solves a few bugs and speeds up hot-reloads a bit.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,32 +2,35 @@
...
     "build-server": "babel src/server/ -d build/server/",
-    "build-electron": "babel src/app/ -d build/app/",
     "build-utilities": "babel src/utilities/ -d build/utilities/",
     "build": " yarn build-ui && yarn build-server && yarn build-utilities && yarn build-electron",
-    "dev": "yarn build-electron && concurrently \"webpack-dev-server --hotOnly\" \"yarn electron-dev\" \"yarn server-dev\"",
-    "server-dev": "yarn build-server && yarn build-utilities && nodemon  --watch src/server/ --watch src/utilities --exec yarn server --ignore build",
     "electron-dev": "NODE_ENV=development electron .",
-    "dist": "electron-builder",
+    "server-dev": "nodemon --watch src/server --watch src/utilities --exec \"yarn server\"",
+    "dev": "yarn build-electron && concurrently \"webpack-dev-server --hotOnly\" \"yarn electron-dev\" \"x-terminal-emulator -e  yarn server-dev\"",
```
Comment (joshxg) [inline package.json]: This script throws an error:
/bin/sh: x-terminal-emulator: command not found 
using mac osx

**Comment by joshxg** [comment]
While we're editing package.json, can we remove the unused dependencies.
- jquery, popper.js & bootstrap

**Comment by wlthomson** [comment]
> While we're editing package.json, can we remove the unused dependencies.
> 
>     * jquery, popper.js & bootstrap

Yep, this is on my TODO list for this PR!

**Comment by joshxg** [comment]
Awesome, thanks. Can create a new issue for terminal creation for logging maybe?

**Pull request merged**